blocks/blocks.go

```diff
@@ -19,48 +20,61 @@ type Block interface {
 	Loggable() map[string]interface{}
 }
 
-// Block is a singular block of data in ipfs
-type RawBlock struct {
+type BasicBlock struct {
```

Comment:
@whyrusleeping, this rename is only in one file, it is exported because I might eventually refactor my Filestore specific changes to blocks/blocks.go into another file.